app/serializers/subject_set_serializer.rb

```diff
@@ -2,9 +2,14 @@ class SubjectSetSerializer
   include RestPack::Serializer
   include FilterHasMany
 
-  attributes :id, :display_name, :set_member_subjects_count, :metadata,
-    :created_at, :updated_at, :href
+  attributes :id, :display_name, :set_member_subjects_count,
+    :retired_set_member_subjects_count, :metadata, :created_at, :updated_at,
+    :href
   can_include :project, :workflows
   can_sort_by :display_name
   can_filter_by :display_name
+
+  def retired_set_member_subjects_count
+    SubjectWorkflowCount.by_set(@model.id).retired.count
```

Comment:
Don't think it's an edge-case, if you have two workflows that share the same subject sets you'll notice this immediately. I stand by my previous suggestion in #986 that this should probably be a hash from either `{workflow_id: retired_count}` in a serialized SubjectSet, or `{subject_set_id: retired_count}` in a serialized Workflow.

/cc @rogerhutchings